prime-router/metadata/fhir_transforms/senders/default-sender-transform.yml

```diff
@@ -9,7 +9,7 @@ elements:
     condition: true
     resource: 'Bundle.entry.resource.ofType(MessageHeader)'
     bundleProperty: '%resource.source.name'
-    value: [ '"CDC PRIME - Atlanta: 2.16.840.1.114222.4.1.237821"' ]
```

Comment:
the OID is not needed here. It is located in the next element `sending-application-universal-id`